samples/storage_sample/uploads_test.py

```diff
@@ -37,8 +37,8 @@ def tearDown(self):
 
   def __ResetUpload(self, size, auto_transfer=True):
     self.__content = ''.join(
-        random.choice(string.letters) for _ in xrange(size))
-    self.__buffer = StringIO.StringIO(self.__content)
+        random.choice(string.ascii_letters) for _ in xrange(size))
```

Comment:
Done. Also got removed a docstring from a unit test case since nose picks up the wrong name in those cases.